docs/advanced/api-v2/rest.md

```diff
@@ -0,0 +1,4019 @@
+---
+title: REST API V2
+---
+
+FORMAT: 1A
+HOST: https://api.counterparty.io
+
+# Counterparty Core API
+
+The Counterparty Core API is the recommended (and only supported) way to query the state of a Counterparty node. 
+
+API routes are divided into 11 groups:
+- [`/blocks`](#group-blocks)
+- [`/transactions`](#group-transactions)
+- [`/addresses`](#group-addresses)
+- [`/assets`](#group-assets)
+- [`/orders`](#group-orders)
+- [`/bets`](#group-bets)
+- [`/dispensers`](#group-dispensers)
+- [`/burns`](#group-burns)
+- [`/events`](#group-events)
+- [`/mempool`](#group-mempool)
+- [`/backend`](#group-backend)
```

Comment:
I think we should change this to `/bitcoin/.

Reply:
done!